proprietary/Configure-TI-CC2541-compatible-proprietary-PHY.md

```diff
@@ -0,0 +1,134 @@
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/ms2692a-signalanalyzers.png">  
+</div> 
+
+
+## 2.2. Software Requirement
+**Simplicity Studio** is a free Eclipse-based Integrated Development Environment (IDE) and a collection of value-add tools provided by Silicon Labs. Developers can use Simplicity Studio to develop, debug and analyze their applications. Silicon Labs recently released the [Simplicity Studio 5](https://www.silabs.com/products/development-tools/software/simplicity-studio/simplicity-studio-5) which is based on the latest versions of Eclipse and the C/C++ Development Tooling. Builds off the features of the previous version and includes advanced SecureVault technology, a new web-like interface, and solid performance improvements.  
+
+**SmartRF™ Studio** SmartRF™ is a Windows application that helps designers of RF systems to easily evaluate the radio at an early stage in the design process for CC2xxx low-power RF devices. It simplifies generation of the configuration register values and commands, as well as practical testing and debugging of the RF system. We use [SmartRF Studio 7](https://www.ti.com/tool/SMARTRFTM-STUDIO).
+
+## 2.3. Given parameter
+Some are provided and some are get from the [xml file](files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/2M_BS-TX_2.xml), finally got below known parameter.
+* Modulation: 2GFSK
+* Datarate: 2Mbps
+* Deviation 500KHz
+* Preamble: 10101010 (13 bytes)
+* Syncword: 0x29417671
+* Payload: 64 bytes variable length
+* Endian: MSB_FIRST
+* CRC polynomial: CCITT_16(0x1021)
+* CRC seed: 0xFFFF
+* Whitening polynomial: PN9_BYTE
+* Whitening seed: 0xFF
+
+# 3. PHY Confirm
+Try to configure a PHY according to above known parameter. The PHY don't work, don't even detect the preamble, so it is worth to annalyze data transmit from CC2541.
+
+## 3.1. Transmit data by SmartRF Studio 7
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/TICC2541-TX.gif">  
+</div> 
+For easier figure out the frame, it is worth to disable the whitening and set some special data bit on the frame, like add 0x00/0xFF.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/Frame.png">  
+</div> 
+
+## 3.2. Capture data on MS2692A
+To get a readable frame data on MS2692A, we need to configure the frequency, reference power level etc. accordingly.
+### 3.2.1 Use "Power vs Time" trace mode to get the TX pluse.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-01-pluse.png">  
+</div> 
+
+### 3.2.2 Change the "Start Time" to locate one pluse.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-02-location.png">  
+</div> 
+
+### 3.2.3 Zoom in.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-03-zoom-in.png">  
+</div> 
```

Comment:
For investigating the packet length, i think you can merge above 3 chapters into 1, and it is OK to keep the last picture.

Reply:
I'd like to keep 3 chapters, show people step by step.